### ISSUE TITLE
`exclude` config parameter does not work

### ISSUE THREAD

**Issue by tmlmt** [harlan-zw/nuxt-simple-sitemap#164]
### Describe the bug

In an i18n case, with a page "/somePage", `exclude: ['/somePage']` or `exclude: ['/es/somePage']` configs don't have any effect, and the page still ends up in the individual sitemaps (e.g. `es-ES-sitemap.xml`)

### Reproduction

https://stackblitz.com/edit/github-vty3pl-txnly4?file=nuxt.config.ts

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Linux
- Node Version:     v18.18.0
- Nuxt Version:     3.8.1
- CLI Version:      3.9.1
- Nitro Version:    2.7.2
- Package Manager:  npm@9.4.2
- Builder:          -
- User Config:      devtools, modules, content, i18n, routeRules, site, sitemap
- Runtime Modules:  @nuxtjs/i18n@8.0.0-rc.5, @nuxt/content@2.8.5, nuxt-simple-sitemap@4.0.1
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Thanks, fixed in v4.1

**Comment by tmlmt** [comment]
Thanks for looking at this. I forked the repro and upgrade to v4.1.0: 
`exclude: ["/somePage"]` is not prefixed in the locale sitemaps exclude: 
```
"sitemaps": {
    "index": {
      "sitemapName": "index",
      "_route": "sitemap_index.xml",
      "sitemaps": [],
      "sources": []
    },
    "en-US": {
      "include": [],
      "exclude": [
        "/somePage",
        "/_nuxt/**",
        "/api/**"
      ],
      "includeAppSources": true,
      "sitemapName": "en-US",
      "_route": "en-US-sitemap.xml",
      "_hasSourceChunk": false,
      "sources": []
    },
    "es-ES": {
      "include": [],
      "exclude": [
        "/somePage",
        "/_nuxt/**",
        "/api/**"
      ],
      "includeAppSources": true,
      "sitemapName": "es-ES",
      "_route": "es-ES-sitemap.xml",
      "_hasSourceChunk": false,
      "sources": []
    },
    "da-DK": {
      "include": [],
      "exclude": [
        "/somePage",
        "/_nuxt/**",
        "/api/**"
      ],
      "includeAppSources": true,
      "sitemapName": "da-DK",
      "_route": "da-DK-sitemap.xml",
      "_hasSourceChunk": false,
      "sources": []
    }
```
So /<locale>/somePage ends up in the individual sitemaps. 
For a prefix_and_default strategy, both /somePage and /<defaultLocale>/somePage should be excluded from the default locale sitemap

**Comment by harlan-zw** [comment]
Thanks, should be fixed in v4.1.1
